diff -Naur xvidcap-1.1.7.orig/configure xvidcap-1.1.7/configure
--- xvidcap-1.1.7.orig/configure	2013-08-21 15:01:45.003756701 +0400
+++ xvidcap-1.1.7/configure	2013-08-21 15:01:53.569756330 +0400
@@ -7913,7 +7913,7 @@
 _ACEOF
 
 
-ALL_LINGUAS="de en it es"
+ALL_LINGUAS="de en it es ru"
 # On IRIX 5.3, sys/types and inttypes.h are conflicting.
 
 
diff -Naur xvidcap-1.1.7.orig/configure.in xvidcap-1.1.7/configure.in
--- xvidcap-1.1.7.orig/configure.in	2013-08-21 15:01:44.847756708 +0400
+++ xvidcap-1.1.7/configure.in	2013-08-21 15:01:50.026756484 +0400
@@ -68,7 +68,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [GETTEXT package name])
 
-ALL_LINGUAS="de en it es"
+ALL_LINGUAS="de en it es ru"
 AM_GLIB_GNU_GETTEXT
 #########################################################
 #########################################################
